traefik/templates/deployment.yaml

```diff
@@ -52,6 +52,9 @@ spec:
       serviceAccountName: {{ include "traefik.serviceAccountName" . }}
       terminationGracePeriodSeconds: 60
       hostNetwork: {{ .Values.hostNetwork }}
+      {{- with .Values.deployment.dnsPolicy }}
```

Comment:
no values for that?

Reply:
Hi, default is we don't need to input any config for `dnsPolicy`

Adding `dnsPolicy: ClusterFirst` might break old deployments

That's why I made it as optional configurable

Reply:
which is fine, but I think we should have at least an example in the `values.yml` even if its commented out. Don't you think so?

Reply:
@SantoDE Absolutely agree. Added example on `values.yml`